2_proxima/0_run-serial-proxima.py

```diff
@@ -63,6 +63,7 @@
     group.add_argument('--error-history', type=int, default=8, help='Number of past timesteps to use to inform controller')
     group.add_argument('--retrain-freq', type=int, default=8, help='How frequently to retrain the evaluator')
     group.add_argument('--min-target-frac', type=float, default=0.05, help='Minimum fraction of evaluations to use DFT')
+    group.add_argument('--n_blending_steps', type=int, default=0, help='How many steps to smoothly blend surrogate and DFT forces')
```

Comment:
Would you mind switching to "-" and not "_" between words? My inner pedantic hobgoblin would thank you

Reply:
I actually already did this but forgot to commit it :laughing: